packages/shipjs/src/step/setup/askQuestions.js

```diff
@@ -73,19 +73,19 @@ async function askBranches(dir) {
 
 async function askCI() {
   const choices = integrations.map(config => config.name);
-  const { CITypeText } = await inquirer.prompt([
+  const { ciTypeText } = await inquirer.prompt([
     {
       type: 'list',
-      name: 'CITypeText',
+      name: 'ciTypeText',
       message: 'Which CI configure?',
       choices,
     },
   ]);
 
-  const CIIndex = choices.indexOf(CITypeText);
-  const CIConfig = await integrations[CIIndex].askQustions();
+  const ciIntegration = choices.indexOf(ciTypeText);
+  const ciConfig = await integrations[ciIntegration].askQuestions();
```

Comment:
Oh sorry, I wasn't clear.
I meant to replace

```
await integrations[CIIndex].askQuestions();
```

with

```
await ciIntegration.askQuestions();
```

Reply:
yes, now it much more clear

Reply:
niceee 👍